File: 217-ContainsDuplicate/217-ContainsDuplicate.java
// Last updated: 6/22/2025, 11:04:23 AM
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for(int i=1;i<nums.length;i++){ 
            if (nums[i]==nums[i-1]){ 
                return true;
            }
            
        }
        return false;
    }
}

File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 6/22/2025, 11:04:27 AM
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        # nums.sort()
        # n=len(nums)
        # for i in range(1,n):
        #     if nums[i]==nums[i-1]:
        #         return True
        # return False
        # if len(nums)!=len(set(nums)):
        #     return True
        # return False
        seen = set()
        for num in nums:
            if num in seen:
                return True
            seen.add(num)
        return False


        